lib/backpex/live_resource.ex

```diff
@@ -1372,6 +1385,8 @@ defmodule Backpex.LiveResource do
 
   def filter_options(_no_filters_present, _filter_configs), do: %{}
 
+  def get_empty_filter_key, do: @empty_filter_key
```

Comment:
```suggestion
  def get_empty_filter_key, do: :empty_filter
```

What's the benefit of using a module attribute here? I recently learned that module attributes should not be used for constants. Do we need this key at compile time?

See also `permitted_order_directions`

https://hexdocs.pm/elixir/module-attributes.html#as-compile-time-constants